src/components/Profile/ClientProfile.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+// Material UI Imports
+import Avatar from '@mui/material/Avatar';
+import BackspaceIcon from '@mui/icons-material/Backspace';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+/**
+ * @typedef {import("../../typedefs.js").clientProfileProps} clientProfileProps
+ */
+
+/**
+ * The ClientProfile Component used to generate the profile card for the client
+ *
+ * @memberof Profile
+ * @name ClientProfile
+ * @param {clientProfileProps} Props - Props for ClientProfile component
+ * @returns {React.JSX.Element} The ClientProfile Component
+ */
+const ClientProfile = ({ clientProfile }) => (
+  <>
+    <Link to="/clients" style={{ textDecoration: 'none', color: 'white' }}>
+      <Button
+        variant="contained"
+        color="secondary"
+        aria-label="Back Button"
+        startIcon={<BackspaceIcon />}
+        /* Temporary solution to clear Documents List after removing permissions */
```

Comment:
Still very confused as to why I'm getting 200 OK responses from getSolidDataset even after permissions revoked.

https://github.com/codeforpdx/PASS/assets/14917816/910168c4-e579-4a0f-8568-cbce80962716

Only seem to get 403 Forbidden back for the documents list if browser refreshes.

Reply:
Permissions isn't the topic of this PR. We can handle this somewhere else.

Reply:
Think I'll just leave it here then. I'll include a TODO comment with the comment I have there right now.